docs/metrics.md

```diff
@@ -88,6 +96,38 @@ An event fired when the user tries to drag or drop a content into the notepad.
 - `cd5`
 - `cd6`
 - `cd10`
+- `cd11`
+
+#### `new-note`
+Event fired when a user creates a new note.
+- `ec` - `notes`
+- `ea` - `new-note`
+- `el` - one of `in-note` or `list-view` where `in-note` refers to the meatball menu flow and `list-view` refers to the button in the list view
+- `cd10`
+- `cd11`
+
+#### `export`
```

Comment:
https://github.com/mozilla/notes/blob/df3df4784bf0107a5db85ba40668b5a55db7489e/src/sidebar/app/actions.js#L159

Reply:
I actually think we should change it...that way we can filter export type on the `el`

Reply:
great idea!

Reply:
High five